Classes/Controller/Form.php

```diff
@@ -1109,6 +1109,9 @@ protected function init()
 
         $this->gp = $this->utilityFuncs->getMergedGP();
 
+        if (!$this->settings['uniqueFormID']) {
```

Comment:
If I understand this correctly, if 'uniqueFormID' is set, we have the same issue.
So removing this if statement?